src/Application.vala

```diff
@@ -41,6 +41,31 @@ public class Camera.Application : Gtk.Application {
                 main_window.destroy ();
             }
         });
+
+        const string DESKTOP_SCHEMA = "org.freedesktop";
```

Comment:
Due to your changes constructor became too big. Maybe it would be nice to decompose it to several init functions